examples/faster_rcnn/README.md

```diff
@@ -4,7 +4,8 @@
 
 | Training Setting | Evaluation | Reference | ChainerCV |
 |:-:|:-:|:-:|:-:|
-| VOC 2007 trainval | VOC 2007 test|  69.9 mAP [1] | 70.6 mAP |
+| VOC07 trainval | VOC07 test|  69.9 mAP [1] | 70.6 mAP |
+| VOC2007\&2012 trainval | VOC07 test|  73.2 mAP [1] | 74.7 mAP |
```

Comment:
I think adding spaces around `|` is beautiful. `VOC07 test|` -> `VOC07 test |`

Reply:
`VOC07 trainval` and `VOC2007\&2012 trainval` look inconsistent.

Reply:
Thank you for pointing that out.